packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -2045,104 +2045,155 @@ void main() {
+    await tester.pump(const Duration(seconds: 10));
+    expect(getContainerColor(tester), equals(disabledColor));
+
+    await pumpDecorator(hovering: true, enabled: false);
+    expect(getContainerColor(tester), equals(disabledColor));
+    await tester.pump(const Duration(seconds: 10));
+    expect(getContainerColor(tester), equals(disabledColor));
+
+    // Outlined
+    const Color blendedHoverColor = Color(0x43009c00);
+    await pumpDecorator(hovering: false, filled: false);
+    await tester.pumpAndSettle();
+    expect(getBorderColor(tester), equals(enabledBorderColor));
+    await tester.pump(const Duration(seconds: 10));
+    expect(getBorderColor(tester), equals(enabledBorderColor));
+
+    await pumpDecorator(hovering: true, filled: false);
+    expect(getBorderColor(tester), equals(enabledBorderColor));
+    await tester.pump(const Duration(milliseconds: 200));
+    expect(getBorderColor(tester), equals(blendedHoverColor));
+
+    await pumpDecorator(hovering: false, filled: false);
+    expect(getBorderColor(tester), equals(blendedHoverColor));
+    await tester.pump(const Duration(milliseconds: 200));
+    expect(getBorderColor(tester), equals(enabledBorderColor));
+
+    await pumpDecorator(hovering: false, filled: false, enabled: false);
+    expect(getBorderColor(tester), equals(enabledBorderColor));
+    await tester.pump(const Duration(milliseconds: 200));
+    expect(getBorderColor(tester), equals(disabledColor));
+
+    await pumpDecorator(hovering: true, filled: false, enabled: false);
+    expect(getBorderColor(tester), equals(disabledColor));
+    await tester.pump(const Duration(seconds: 10));
+    expect(getBorderColor(tester), equals(disabledColor));
   });
 
   testWidgets('InputDecorator draws and animates focusColor', (WidgetTester tester) async {
-    const Color fillColor = Color(0xFF00FF00);
     const Color focusColor = Color(0xFF0000FF);
-
-    await tester.pumpWidget(
-      buildInputDecorator(
-        isFocused: false,
-        decoration: const InputDecoration(
-          filled: true,
-          fillColor: fillColor,
-          focusColor: focusColor,
+    const Color fillColor = Color(0x0A000000);
+    const Color disabledColor = Color(0x05000000);
+    const Color enabledBorderColor = Color(0x1f000000);
+
+    Future<void> pumpDecorator({bool focused, bool enabled = true, bool filled = true}) async {
+      return await tester.pumpWidget(
+        buildInputDecorator(
+          isFocused: focused,
+          decoration: InputDecoration(
+            enabled: enabled,
+            filled: filled,
+            focusColor: focusColor,
+            focusedBorder: const OutlineInputBorder(borderSide: BorderSide(color: focusColor)),
+            disabledBorder: const OutlineInputBorder(borderSide: BorderSide(color: disabledColor)),
+            border: const OutlineInputBorder(borderSide: BorderSide(color: enabledBorderColor)),
+          ),
         ),
-      ),
-    );
+      );
+    }
 
+    // Filled
```

Comment:
Nit: add period.

```suggestion
    // Filled.
```


https://dart.dev/guides/language/effective-dart/documentation#do-format-comments-like-sentences

Reply:
Done.